src/thx/Arrays.hx

```diff
@@ -622,6 +622,21 @@ It returns the last element of the array or null if the array is empty.
     return array[array.length-1];
 
 /**
+Static wrapper for `Array` `map` function.
+**/
+  #if js inline #end
```

Comment:
I would not specialize for `JS`. Native `map` is more likely to be slower that this map.